### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by sigma1326**
With this update, the #34 and #35 issues can be closed. Moreover, some files are reformatted and some other codes are refactored.

### CODE REVIEW

File: tase/db/arangodb/graph/vertices/hit.py [hunk 1]
```diff
@@ -90,8 +91,8 @@ def create_hit(
 
         Raises
         ------
-        Exception
-            If could not create the `has` edge from `Hit` vertex to `Audio` vertex
+        EdgeCreationFailed
+            If creation o the `has` edge from `Hit` vertex to `Audio` vertex
```
Comment (sigma1326) [inline tase/db/arangodb/graph/vertices/hit.py:95]: There is a grammar mistake after the `creation` word

**Review by sigma1326 — COMMENTED** [review]
I double-checked these changes. Except for a few grammar mistakes that I fixed, everything looks good.

**Review by soran-ghaderi — COMMENTED** [review]
To many files changed without tests and therefore difficult to correctly verify. 

I noticed that the majority of them were package importing so briefly checked other changes. Make sure they're all working.

**Comment by soran-ghaderi** [comment]
> I double-checked these changes. Except for a few grammar mistakes that I fixed, everything looks good.

Ok then.

**Review by soran-ghaderi — APPROVED** [review]

**Pull request merged**